components/scream/src/physics/p3/p3_functions.hpp

```diff
@@ -246,21 +246,24 @@ struct Functions
   // Calls polysvp1 to obtain the saturation vapor pressure, and then computes
   // and returns the saturation mixing ratio, with respect to either liquid or ice,
   // depending on value of 'ice'
-  KOKKOS_INLINE_FUNCTION
+  KOKKOS_FUNCTION
   static Spack qv_sat(const Spack& t_atm, const Spack& p_atm, const bool ice);
 
   // TODO: comment
-  KOKKOS_INLINE_FUNCTION
+  KOKKOS_FUNCTION
   static void get_cloud_dsd2(
     const Smask& qc_gt_small, const Spack& qc, Spack& nc, Spack& mu_c, const Spack& rho, Spack& nu,
     const view_1d<const Scalar>& dnu, Spack& lamc, Spack& cdist, Spack& cdist1, const Spack& lcldm);
 
   // Computes and returns rain size distribution parameters
-  KOKKOS_INLINE_FUNCTION
+  KOKKOS_FUNCTION
   static void get_rain_dsd2 (
     const Smask& qr_gt_small, const Spack& qr, Spack& nr, Spack& mu_r,
     Spack& lamr, Spack& cdistr, Spack& logn0r, const Spack& rcldm);
```

Comment:
Switching from KOKKOS_INLINE_FUNCTION to KOKKOS_FUNCTION fixed a warning thrown by GCC.